api/db/index.js

```diff
@@ -75,6 +77,7 @@ const setup = (
 };
 
 module.exports = {
-  setup,
-  models: exportedModels
+  models: exportedModels,
+  raw: (...args) => (knexObject ? knexObject(...args) : null),
```

Comment:
Exposes the knex object directly because it can be a lot easier to manually create some of our data relationships than to use Bookshelf.  :sigh:

Reply:
i understand :)